VimR/VimR/PrefMiddleware.swift

```diff
@@ -11,7 +11,7 @@ final class PrefMiddleware: MiddlewareType {
   typealias StateType = AppState
   typealias ActionType = AnyAction
 
-  static let compatibleVersion = "168"
+  static let compatibleVersion = "169"
```

Comment:
Since the new field nvimBinary is set to default, we don't have to increase the pref version.

Reply:
done!